src/SFA.DAS.CaptureTheFlag.Web/Data/ApplicationUser.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Microsoft.AspNetCore.Identity;
+
+namespace DAS_Capture_The_Flag.Data
+{
+    public class ApplicationUser : IdentityUser
```

Comment:
Extending the IdentityUser is a good strategy for future properties that may be needed (Open/Closed principle)